iina/AppDelegate.swift

```diff
@@ -207,12 +207,16 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }
   }
 
-  private func showWelcomeWindow() {
+  private func showWelcomeWindow(checkingForUpdatedData: Bool = false) {
```

Comment:
The ```showWelcomeWindow()``` call in ```applicationShouldHandleReopen``` should set this parameter to ```true```.

Reply:
Fixed by acc6e511.